specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-03-01/capacityReservation.json

```diff
@@ -361,6 +361,22 @@
               "name": "ExpandTypesForGetCapacityReservationGroups",
               "modelAsString": true
             }
+          },
+          {
+            "name": "resourceIdsOnly",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The query option to fetch Capacity Reservation Group Resource Ids. 'CreatedInSubscription' enables fetching Resource Ids for all capacity reservation group resources created in the subscription.'SharedWithSubscription' enables fetching Resource Ids for all capacity reservation group resources shared with the subscription.'All' enables fetching Resource Ids for all capacity reservation group resources shared with the subscription and created in the subscription.",
```

Comment:
Add a break line between the explanation for each resource Id option.

Reply:
updated